.github/codeql/paths.yml

```diff
@@ -0,0 +1,7 @@
+paths-ignore:
+  - cypress
+  - test
+  - testFixtures
+  - src/lib/config.ts
+  - src/pages/login/reset-password.tsx
+  - src/useCases/*

```

Comment:
Should we really be ignoring all of the `useCases`? I'd say this is actually where the bulk of the code for the service is?

Reply:
Ok, on further inspection, it turns out my intuition was wrong; the useCases only actually represent about 30% of the files and 9% of the code base:

```
❯ cloc src
     172 text files.
     172 unique files.
       1 file ignored.

github.com/AlDanial/cloc v 1.92  T=0.09 s (1839.2 files/s, 193256.9 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
TypeScript                     169            913             48          16186
JSON                             2              0              0            904
SCSS                             1              3              0             19
-------------------------------------------------------------------------------
SUM:                           172            916             48          17109
-------------------------------------------------------------------------------
```

```
❯ cloc src/useCases
      52 text files.
      52 unique files.
       0 files ignored.

github.com/AlDanial/cloc v 1.92  T=0.03 s (1864.1 files/s, 67000.7 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
TypeScript                      52            270              5           1594
-------------------------------------------------------------------------------
SUM:                            52            270              5           1594
-------------------------------------------------------------------------------
```

...But my question still stands; why are we excluding these?

Reply:
Agreed

Reply:
@daviesjamie this was my assumption that UseCases where tests, hence I asked you guys to go through these ;)